packages/tasks/src/model-libraries.ts

```diff
@@ -686,6 +686,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.whisperkit,
 		countDownloads: `path_filename:"model" AND path_extension:"mil" AND _exists_:"path_prefix"`,
 	},
+	"3dtopia-xl": {
```

Comment:
I don't see any model checkpoints when I look here: https://huggingface.co/models?other=3dtopia-xl

It might be that your model checkpoints are private, do make sure to make them public before asking for a final review! 🤗

Reply:
It is now officially released for both the huggingface repo and github repo, as well as model checkpoints. Kindly let me know if there is additional material needed from my side. Thanks!